bigquery/tests/test_query_builder.py

```diff
@@ -52,6 +86,63 @@ def test_no_selects(self):
 
         self.assertEqual(result, 'SELECT *')
 
+    def test_aggregation_within_record(self):
```

Comment:
These're great tests, which is super important given the amount of string manipulation that we're doing